### PR TITLE
Security nav updates

### PR DESCRIPTION
**Pull request by sowasred2012**
## Done

**[Livepatch](https://ubuntu.com/security/livepatch)**
- Added section linking to livepatch docs - [copy doc](https://docs.google.com/document/d/1g_Ip5MzlBRpbM2wULpSnsR5XaHJVWgLPLdq8_7fCHrs/edit#)

**[Certifications & Hardening](https://ubuntu.com/security/certifications)**
- Added a section linking to docs
- Linked to docs in header
- Added cards linking to subpages - [copy doc](https://docs.google.com/document/d/13RcVvHBAoywfk3o5Dy-16F3KrMytL9-2v6wWbW444Ko/edit#)

**[CVEs](https://ubuntu.com/security/cves)**
- Added description of CVEs and a link to OVAL - [copy doc](https://docs.google.com/document/d/1IJyzz5hi90UJwzJ4zJuVLpLAFEvqgreSAPasJvluaAs/edit#)

**[Notices](https://ubuntu.com/security/notices)**
- Added link to OVAL - [copy doc](https://docs.google.com/document/d/1d9gXYWsAE_ktvPQ5cO0yMA9YrS6MW0xvhAsoqUFLj-I/edit#)

## QA

- Visit https://ubuntu-com-11757.demos.haus/security/livepatch
- See that it matches the [copy doc](https://docs.google.com/document/d/1g_Ip5MzlBRpbM2wULpSnsR5XaHJVWgLPLdq8_7fCHrs/edit#)
- Visit https://ubuntu-com-11757.demos.haus/security/certifications
- See that it matches the [copy doc](https://docs.google.com/document/d/13RcVvHBAoywfk3o5Dy-16F3KrMytL9-2v6wWbW444Ko/edit#)
- Visit https://ubuntu-com-11757.demos.haus/security/cves
- See that it matches the [copy doc](https://docs.google.com/document/d/1IJyzz5hi90UJwzJ4zJuVLpLAFEvqgreSAPasJvluaAs/edit#)
- Visit https://ubuntu-com-11757.demos.haus/security/notices
- See that it matches the [copy doc](https://docs.google.com/document/d/1d9gXYWsAE_ktvPQ5cO0yMA9YrS6MW0xvhAsoqUFLj-I/edit#)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/5241

### CODE REVIEW

File: templates/security/livepatch.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
 meta_copydoc %}
 
 {% block content %}
-<section class="p-strip u-no-padding--top u-no-padding--bottom is-bordered">
+<section class="p-strip u-no-padding--top is-bordered">
   <div class="p-strip--suru-topped">
     <div class="row u-equal-height">
       <div class="col-7">
```
Comment (petesfrench) [inline templates/security/livepatch.html:15]: Should we hide the video on smaller screens or is it something we want to keep visible?

File: templates/security/livepatch.html [hunk 2]
```diff
@@ -29,6 +29,28 @@ <h2 class="p-heading--4">Reduce downtime and unplanned work</h2>
       </div>
     </div>
   </div>
+
+  <div class="u-fixed-width">
+    <hr class="p-separator" />
+  </div>
+
+  <div class="row">
+    <div class="col-4">
```
[the remark this answers is not in the document]
Comment (petesfrench) [reply templates/security/livepatch.html:38]: There are two other images down the page that need u-hide--small&medium. As well the 'Dena' logo needs a u-algin--center

**Comment by davegoddard42** [comment]
From a design POV - all good, except the livepatch docs strip - could we make it more like the below?

Thanks!

image

File: templates/security/certifications.html [hunk 3]
```diff
@@ -202,6 +233,26 @@ <h3>How do I comply with PCI-DSS?</h3>
...
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-4">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/5865c9d2-image-document-ubuntuterms.png",
+        alt="",
+        width="166",
+        height="200",
+        hi_def=True,
+        loading="auto"
```
Comment (petesfrench) [inline templates/security/certifications.html:245]:
```suggestion
        loading="lazy"
```

**Comment by petesfrench** [comment]
Outside the scope of these updates, on /security/cves, the text 'Recent CVEs affecting Ubuntu' appears even when there are no results:
image

**Review by petesfrench — CHANGES_REQUESTED** [review]
A couple little changes, then it should be good to go :+1:

File: templates/security/livepatch.html (hunk 1 above)
Comment (sowasred2012) [reply templates/security/livepatch.html:15]: We should keep it visible IMO

**Comment by davegoddard42** [comment]
> * https://ubuntu-com-11757.demos.haus/security/livepatch

@petesfrench - I can't replicate - for me I see "0 results". 
image

**Comment by davegoddard42** [comment]
@sowasred2012 - better, but can we change the CTA to a link, style the H2 so it looks like a H3, and for the image, set the width to 100 and put it across only 2 columns, not 4?

**Comment by petesfrench** [comment]
@davegoddard42 It appears on https://ubuntu-com-11757.demos.haus/security/cves

**Review by petesfrench — APPROVED** [review]
LGTM

**Comment by davegoddard42** [comment]
> @davegoddard42 It appears on https://ubuntu-com-11757.demos.haus/security/cves

Ah ok, looks like it's just the demo affected then? It's fine on what's live.

**Pull request merged**